### ISSUE TITLE
Package install failure

### ISSUE THREAD

**Issue by invertdna** [statdivlab/paramedic#16]
Apologies if this is a me problem... but I'm getting an install error that looks like a problem w the Stan headers: 
```
In file included from /Library/Frameworks/R.framework/Versions/4.3-x86_64/Resources/library/StanHeaders/include/stan/math/prim/core.hpp:4:
/Library/Frameworks/R.framework/Versions/4.3-x86_64/Resources/library/StanHeaders/include/stan/math/prim/core/init_threadpool_tbb.hpp:12:10: fatal error: 'tbb/task_scheduler_init.h' file not found
#include <tbb/task_scheduler_init.h>
```
It seems like there are similar issues w this `.h` file elsewhere: https://github.com/stan-dev/rstan/issues/924 , for what it's worth. 

As is in the path above, I'm running R 4.3 (June 2023), on an Intel Mac (OS 14.1.2)

**Comment by adw96** [comment]
@bdwilliamson Hi Brian! Are you able to replicate @invertdna 's issue? Lmk if you don't have spare bandwidth and I'll see what I can do. Hope you're doing really well!!!

**Comment by bdwilliamson** [comment]
This should be fixed in v0.1.3.6. Thanks for pointing me to the rstan issue -- an update to how StanHeaders works required us to add some lines to the install code.

**Comment by adw96** [comment]
Thank you so much, @bdwilliamson !! I really appreciate your help. 

@invertdna Feel free to reopen if this fix doesn't work, and thanks again for your interest in the package.
